lib/matplotlib/cbook/__init__.py

```diff
@@ -1044,6 +1044,69 @@ def delete_masked_points(*args):
     return margs
 
 
+def _combine_masks(*args):
+    """
+    Find all masked and/or non-finite points in a set of arguments,
+    and return the arguments as masked arrays with a common mask.
+
+    Arguments can be in any of 5 categories:
+
+    1) 1-D masked arrays
+    2) 1-D ndarrays
+    3) ndarrays with more than one dimension
```

Comment:
Please indicate that the intent is to support RGB triplet arrays and that we don't expect invalid values there (as discussed in the call), to explain the behavior.

Reply:
Done.